src/core/operator_list.js

```diff
@@ -487,11 +507,20 @@ class QueueOptimizer extends NullOptimizer {
       iCurr: 0,
       fnArray: queue.fnArray,
       argsArray: queue.argsArray,
+      isOffscreenCanvasSupported: false,
     };
     this.match = null;
     this.lastProcessed = 0;
   }
 
+  get isOffscreenCanvasSupported() {
+    return this.context.isOffscreenCanvasSupported;
+  }
+
+  set isOffscreenCanvasSupported(value) {
```

Comment:
Is the getter actually used for anything? Otherwise you want the following instead.
```suggestion
  // eslint-disable-next-line accessor-pairs
  set isOffscreenCanvasSupported(value) {
```